### PR TITLE
Signup fixed

### PR DESCRIPTION
**Pull request by Heba-Elzaher**

### CODE REVIEW

**Review by ahmadaljaghbeir — APPROVED** [review]

File: src/Components/LandPage/SignUp.jsx [hunk 1]
```diff
@@ -1,52 +1,141 @@
...
+                                        <Input size="md" placeholder="Additional Note..." />
+                                    </Input.Wrapper>
+                                </Grid.Col>
+                            </Grid>
                         </Stepper.Step>
                         <Stepper.Completed>
                             <Center>
                                 <Title order={2}>Completed, account has been created successfully!</Title>
                             </Center>
-
```
Comment (ahmadaljaghbeir) [inline src/Components/LandPage/SignUp.jsx:51]: delete

File: src/Components/LandPage/SignUp.jsx [hunk 2]
```diff
@@ -1,52 +1,141 @@
...
+                                        placeholder="Password.."
+                                        visible={visible}
+                                        onVisibilityChange={toggle}
+                                        withAsterisk
+                                    />
+                                </Grid.Col>
+                            </Grid>
                         </Stepper.Step>
                         <Stepper.Step label="Final step" description="Health Information">
-                            <SignUpForm label1="Insurance type:" placeholder1="Insurance type.."
```
Comment (ahmadaljaghbeir) [inline src/Components/LandPage/SignUp.jsx:40]: delete

File: src/Components/LandPage/SignUp.jsx [hunk 3]
```diff
@@ -1,52 +1,141 @@
...
-                                label2="Blood Type:" placeholder2="Blood type.."
```
Comment (ahmadaljaghbeir) [inline src/Components/LandPage/SignUp.jsx:41]: delete

File: src/Components/LandPage/SignUp.jsx [hunk 4]
```diff
@@ -1,52 +1,141 @@
...
-                            />
-                            <SignUpForm label1="Allergies:" placeholder1="Allergies.."
```
Comment (ahmadaljaghbeir) [inline src/Components/LandPage/SignUp.jsx:43]: delete

File: src/Components/LandPage/SignUp.jsx [hunk 5]
```diff
@@ -1,52 +1,141 @@
...
+                                            <option value="1">None</option>
+                                            <option value="2">Male</option>
+                                            <option value="3">Female</option>
+                                        </Input>
+                                    </Input.Wrapper>
+                                </Grid.Col>
+                            </Grid>
                         </Stepper.Step>
                         <Stepper.Step label="Second step" description="Create Password">
-                            <SignUpForm label1="Password" placeholder1="Password"
```
Comment (ahmadaljaghbeir) [inline src/Components/LandPage/SignUp.jsx:35]: delete

File: src/Components/LandPage/SignUp.jsx [hunk 6]
```diff
@@ -1,52 +1,141 @@
...
 
     return (
         <>
-            <Grid mt={100}>
+            <Grid mt={150}>
                 <Grid.Col span={3}></Grid.Col>
                 <Grid.Col span={6}>
                     <Stepper active={active}>
                         <Stepper.Step label="First step" description="Patient Information">
-                            <Stack>
```
Comment (ahmadaljaghbeir) [inline src/Components/LandPage/SignUp.jsx:19]: delete

File: src/Components/LandPage/SignUp.jsx [hunk 7]
```diff
@@ -1,52 +1,141 @@
 import { useState } from 'react';
-import { Stepper, Button, Group, Grid, Center, Stack, Title } from '@mantine/core';
```
Comment (ahmadaljaghbeir) [inline src/Components/LandPage/SignUp.jsx:2]: delete

**Comment by Moutasim02** [comment]
@Heba-Elzaher
This fixes some stuff but still breaks the mobile view somehow, The one in main branch is working well with mobile, try to edit the fields directly without changing the whole components.
image

**Pull request merged**